integration/client_server_test.go

```diff
@@ -622,10 +621,14 @@ func setup(t *testing.T, options setupOptions) (string, string) {
 		osArgs := setupServer(addr, options.token, options.tokenHeader, options.pathPrefix, cacheDir, options.cacheBackend)
 
 		// Run Trivy server
-		require.NoError(t, execute(osArgs))
+		err := execute(osArgs)
+		if err != nil {
+			panic(err)
```

Comment:
nit:
IIUC `t.Error` and `required.NoError` do not stop the test.
I don't like to use `panic`, but here we use it in tests, so it is not so critical.
But maybe we want to write a small comment.
```suggestion
                        // panic is needed to instantly stop the test (client part)
			panic(err)
```

Reply:
Doesn't `require.NoError` call `t.FailNow` which stops the test execution?

Reply:
This is called in goroutine, and the linter complained that `require` should not be used in non-main threads. I'm also not a fan of panic. I changed it to `assert`, although it's not ideas as it cannot halt the test.
https://github.com/aquasecurity/trivy/pull/9060/commits/8708a06bbbc76cffa8ced7cdb6365f98758fa8f8